test/uk/gov/hmrc/cgtpropertydisposalsfrontend/services/onboarding/SubscriptionServiceImplSpec.scala

```diff
@@ -371,31 +365,45 @@ class SubscriptionServiceImplSpec extends WordSpec with Matchers with MockFactor
         ManuallyEnteredContactName
       )
 
-      "user is successfully registered and subscribed returning correct cgt reference" in {
-        val registrationResponse = Json.parse(
-          s"""
-             |{
-             |  "sapNumber" : "$sapNumber"
-             |}
-             |""".stripMargin
+      "successfully register and subscribed returning correct cgt reference" in {
+        mockRegisterWithoutId(registrationDetails)(
```

Comment:
using `inSequence` like:
```scala
inSequence{
  mockRegisterWithoutId(...)
  mockSubscribe(..)
}
```
will make this test stronger as it will assert that the things happen in that order